channels/ru.m3u

```diff
@@ -880,7 +880,7 @@ http://1hdru-hls-otcnet.cdnvideo.ru/siesta/tracks-v1a1/index.m3u8
 http://62.32.67.187:1935/WEB_Smilik/ngrp:Smilik.stream-adaptive/playlist.m3u8
 #EXTINF:-1 tvg-id="SmaylikTV.ru" tvg-name="Смайлик ТВ" tvg-country="RU" tvg-language="Russian" tvg-logo="http://smilik.tv/wp-content/uploads/2016/02/Desktop_-1.png" group-title="Kids",Смайлик ТВ (720p)
 http://62.32.67.187:1935/WEB_Smilik/Smilik.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="SportTV.ru" tvg-name="Sport TV" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Sports",Sport ТV [No Sound]
+#EXTINF:-1 tvg-id="SportTV.ru" tvg-name="Sport TV" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Sports",Sport ТV
```

Comment:
What a mess, where is it from? Almost convinced myself it's a Polish channel but now it shows a single Ukrainian ad 7 times in a row. Absolutely unsigned channel

Reply:
Weird... found it in a random playlist lurking on the Internet.
What's interesting is that they gave him 3 different names... cool to find the actual name (Fight Box, Sport TV, UFC...)

Reply:
That's getting funnier. Referee speaks in Russian and commentary sounds like an alien

Reply:
![vlcsnap-2021-08-02-01h31m55s682](https://user-images.githubusercontent.com/30985701/127788509-2e9590dc-7bd8-469c-85c3-88ea12f7e576.png)

Reply:
> 
> 
> That's getting funnier. Referee speaks in Russian and commentary sounds like an alien

Lol ! At that state, i'm gonna remove that channel, since we can't seem to put our hands on what it is x)

Reply:
It's 1+1 Спорт from Ukraine, only available here
https://1plus1.video/tvguide/sport/online

Reply:
![image](https://user-images.githubusercontent.com/30985701/127788689-9ca813ca-dc96-44fb-a4fc-c8e5dec79f91.png)
Found that out, we already had it.

Reply:
Seems they got different servers, https://live-fdc07-ams.1plus1.video/live/1627959900/IP2NXOS92aRcPdJ1tUd_hg/sport/smil:sport.smil/playlist.m3u8 is another one of them, given by https://grandcentral.1plus1.video/live/36/3b1b37e842d182c884ef55cd8951cc30/1627959900?return_http=true&_t744940799313

Reply:
That's definitely not НТВ+ as it ceased to exist years ago

Reply:
Interesting...

Well, we shall have a look again at some of the Russian channels then. You'll sort them better than I, I suppose lol